arangod/IResearch/IResearchViewStoredValues.h

```diff
@@ -55,6 +55,12 @@ class IResearchViewStoredValues {
     bool operator!=(StoredColumn const& rhs) const noexcept {
       return !(*this == rhs);
     }
+
+    bool sameName(std::string_view str) const noexcept {
+      return (str.size() + 1) == name.size() && 
+        strncmp(str.data(), name.data() + 1, str.size()) == 0;
```

Comment:
```suggestion
      return std::string_view{name.data(), name.size()} == str;
```
https://godbolt.org/z/9rfE45Wz9

Reply:
The suggested code has different behaviour than original. But as we are on c++20 there is even better way  - `ends_with`